src/leappto/providers/libvirt_provider.py

```diff
@@ -151,11 +151,6 @@ def __inspect_os(domain_name):
 
             return (hostname, Installation(OperatingSystem(distro, '{}.{}'.format(major, minor)), packages))
 
-        def __vagrant_ssh_checkoutput(domain_name, command):
```

Comment:
no it doesn't belong in there ;-) I just haven't moved it out of here just yet but will do